presto-product-tests/conf/docker/fileauth/compose.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
I would love to migrate these tests as regular unit (integration) tests. There you could test more scenarios and they are much easier to debug.

Reply:
@kokosing I have added both UTs and product tests. Do you want me to add anything specific?